src/plugins/data_source/common/index.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
```

Comment:
I would suggest to use new license header: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/.eslintrc.js#L31-L55

Reply:
updated license header with a new commit